<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReminderDialog</class>
 <widget class="QDialog" name="ReminderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>75</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Reminder</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../assets/tomato.png</normaloff>../assets/tomato.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTimeEdit" name="reminderTime">
       <property name="displayFormat">
        <string>hh:mm a</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="reminderInput">
       <property name="placeholderText">
        <string>Enter text...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="reminderSubmit">
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
